runtime/compiler/env/JITServerPersistentCHTable.cpp

```diff
@@ -46,7 +46,7 @@ JITServerPersistentCHTable::~JITServerPersistentCHTable()
       {
```

Comment:
Above in the constructor of `JITServerPersistentCHTable` I am thinking that we could be more explicit in 
` _classMap(decltype(_classMap)::allocator_type(TR::Compiler->persistentAllocator()))`
and use the allocator that comes with the persistentMemory given as a parameter. I assume that persistentMemory is the per-client one.

Reply:
Done